.circleci/config.yml

```diff
@@ -0,0 +1,116 @@
+version: 2
+# For more information on choosing an image (or alternatively using a
+# VM instead of a container) see https://circleci.com/docs/2.0/executor-types/
+# To see the list of pre-built images that CircleCI provides for most common languages see
+# https://circleci.com/docs/2.0/circleci-images/
+defaults: &defaults
+  docker:
+    - image: circleci/build-image:ubuntu-14.04-XXL-upstart-1189-5614f37
```

Comment:
Does this image have MySQL on it? We will need that for running the full set of tests...

We did this in workbench like so:

https://github.com/all-of-us/workbench/blob/f8eb2419cb5d3a97c6d9db630e56775689dfaee3/.circleci/config.yml#L46

Reply:
yes it does